test/Nodes/all-cf

```diff
@@ -2,6 +2,6 @@ chap01
 chap05
 chap06
 chap08
-chap12
+chap11
```

Comment:
I think we still want to leave chap11 off the all-cf list. chap11 is still running gasnet testing.